docs/product-migration-guide/src/main/asciidoc/migration-tool-7.0-proc.adoc

```diff
@@ -0,0 +1,158 @@
+
+.Procedure
+. Locate the `bin/.niogit` directory in your current {CENTRAL} distribution for {PRODUCT_OLD} 6.4 and locate the same directory in your new {PRODUCT} {PRODUCT_VERSION} installation.
++
+--
+Example:
+[source]
+----
+$ ~/$JBOSS_HOME/bin/.niogit
+----
+
+ifdef::BA[]
+[source]
+----
+$ ~/$RHBA_HOME/bin/.niogit
+----
+endif::BA[]
+ifdef::DM[]
+[source]
+----
+$ ~/$RHDM_HOME/bin/.niogit
+----
+endif::DM[]
+--
+. In your command terminal, copy the `.niogit` directory from your current {CENTRAL} distribution for {PRODUCT_OLD} 6.4 to the `bin` folder of the new {PRODUCT} {PRODUCT_VERSION} installation directory:
++
+--
+ifdef::BA[]
+[source]
+----
+cp -r /$JBOSS_HOME/bin/.niogit  /$RHBA_HOME/bin/
+----
+endif::BA[]
+ifdef::DM[]
+[source]
+----
+cp -r /$JBOSS_HOME/bin/.niogit  /$RHDM_HOME/bin/
+----
+endif::DM[]
+
+The copied `.niogit` folder is the directory that the migration tool will prepare for {PRODUCT} {PRODUCT_VERSION}. The original `.niogit` folder remains in your {PRODUCT_OLD} 6.4 installation and will not be modified by the migration tool in this procedure.
+--
+. In the {PRODUCT} {PRODUCT_VERSION} `bin` directory, delete the `.index` directory. You must delete this folder because {PRODUCT} {PRODUCT_VERSION} comes with certain updates that are not compatible with {PRODUCT_OLD} 6.4. The `bin` directory will be re-indexed after the migration when you start {PRODUCT} {PRODUCT_VERSION}.
+. Note the path to the copied `.niogit` directory for {PRODUCT} {PRODUCT_VERSION}. This will be required for running the migration tool.
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
+* *Product:* {PRODUCT_SHORT}
+* *Version:* {PRODUCT_VERSION}
+. Download *{PRODUCT} {PRODUCT_VERSION_LONG} Add-Ons* and extract the downloaded `{PRODUCT_FILE}-add-ons.zip` folder to a temporary directory.
+. In the extracted `{PRODUCT_FILE}-add-ons` folder, extract the `{PRODUCT_INIT}-{PRODUCT_VERSION}-migration-tool.zip` sub-folder. This exposes the `kie-wb-common-project-migration-$VERSION` folder.
++
+. In your command terminal, navigate to the temporary directory where you extracted the `kie-wb-common-project-migration-$VERSION` folder and run the following command. The `${PRODUCT_INIT_CAP}_NIOGIT_DIR` is the path to the `.niogit` directory that you previously copied to the {PRODUCT} {PRODUCT_VERSION} installation.
++
+--
+On Linux or UNIX-based systems:
+[source]
+ifdef::BA[]
+----
+$ ~/$INSTALL_DIR/kie-wb-common-project-migration-$VERSION/bin/migration-tool.sh -t $RHBA_NIOGIT_DIR
+----
+endif::BA[]
+ifdef::DM[]
+----
+$ ~/$INSTALL_DIR/kie-wb-common-project-migration-$VERSION/bin/migration-tool.sh -t $RHDM_NIOGIT_DIR
+----
+endif::DM[]
+
+On Windows:
+[source]
+ifdef::BA[]
+----
+$ ~/$INSTALL_DIR/kie-wb-common-project-migration-$VERSION/bin/migration-tool.bat -t $RHBA_NIOGIT_DIR
```

Comment:
For this and the Windows path below: they should probably use `\` slashes, right? IMO it makes sense for all the other paths to stick with `/` slashes (even though technically all these paths are different on Windows), but it seems odd to have an invocation specifically titled "On Windows" with the wrong path type.

Reply:
Definitely, thanks. Done.